src/top/metrics.rs

```diff
@@ -397,6 +397,20 @@ async fn errors_totals(
     }
 }
 
+async fn uptime_changed(client: Arc<SubscriptionClient>, tx: state::EventTx) {
+    tokio::pin! {
+        let stream = client.uptime_subscription();
+    };
+
+    while let Some(Some(res)) = stream.next().await {
+        if let Some(d) = res.data {
+            _ = tx
```

Comment:
nit: **Not to be changed in this PR**

This is a bad pattern which is copied over from the other functions 😕. We really should have some error handling inside `subscribe` and refactor this whole file